src/testdrive/src/action.rs

```diff
@@ -407,12 +419,24 @@ pub fn build(cmds: Vec<PosCommand>, state: &State) -> Result<Vec<PosAction>, Err
                     "s3-add-notifications" => {
                         Box::new(s3::build_add_notifications(builtin).map_err(wrap_err)?)
                     }
+                    "set-regex" => {
```

Comment:
@benesch all the set commands in the same .td file appear to be read together in the beginning, so they take effect for the entire test from the start rather than from the point they are defined to the next point where they are changed. But I guess this is a discussion for another day.